frontend/challedu-app/src/components/navbar.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import { Navbar, NavbarBrand, Nav, NavItem, Button}  from 'reactstrap';
+import '../App.css';
+import logo from '../images/logo.svg';
+
+export default class navbar extends Component {
+    render() {
```

Comment:
no need for class component here, make it functional component

Reply:
i cant understand the difference, can you please explain why this should be functional component?

Reply:
@GMouzakis you should aim for functionnal components as this is what is pushed by react, class components you will use in older projects

Reply:
Thanks for the tip @Pauleing.